spec.md

```diff
@@ -118,6 +118,7 @@ of the delivery domain) can perform such a downgrade or interception attack.
 This document defines a mechanism for recipient domains to publish policies
 specifying:
 
+   * MTA that handles emails for the recipient domain
```

Comment:
I think we need to put this explicitly because this proposed policy actually does two things:
1. Detect MX mismatches
2. Validate certificate.

Validate certificate is applicable only if the MX matches with the one mentioned in the policy. On top of that mentioning the first point also adds more clarity to readers.